src/ctelnet.cpp

```diff
@@ -1670,18 +1670,18 @@ void cTelnet::processTelnetCommand(const std::string& telnetCommand)
 
         if (option == OPT_MSDP) {
             //MSDP support
-            std::string output;
             if (!mpHost->mEnableMSDP) {
-                output += TN_IAC;
-                output += TN_DONT;
-                output += OPT_MSDP; // disable MSDP per http://tintin.sourceforge.net/msdp/
-                // This will be unaffected by Mud Server encoding:
-                socketOutRaw(output);
```

Comment:
Isn't the group of lines being removed here *equivalent* to the `sendTelnetOption(TN_DONT, OPT_MSDP)` anyhow? :confused:

Reply:
Yes, it confirms to the pattern set by the remainder of the case statements below it - this one was an anomaly.